e2e/cypress/support/index.js

```diff
@@ -223,3 +218,13 @@ function sysadminSetup(user) {
         }
     });
 }
+
+function resetUserPreference(userId) {
+    cy.apiSaveTeammateNameDisplayPreference('username');
+    cy.apiSaveLinkPreviewsPreference('true');
+    cy.apiSaveCollapsePreviewsPreference('false');
+    cy.apiSaveClockDisplayModeTo24HourPreference(false);
+    cy.apiSaveTutorialStep(userId, '999');
+    cy.apiSaveOnboardingPreference(userId, 'hide', 'true');
+    cy.apiSaveCloudTrialBannerPreference(userId, '14_days_banner');
```

Comment:
Removed cloud trial banner to not show up whenever admin is used for testing

Reply:
Oh I see. I think you're right. There's another option as "hide". I'll update this PR. Good catch! Thanks!

Reply:
I've updated the command for consistency with API call. Unfortunately, "hide" is not being respected in some case like https://github.com/mattermost/mattermost-webapp/blob/master/components/announcement_bar/cloud_trial_announcement_bar/cloud_trial_announcement_bar.tsx#L140-L144. It's a separate issue to clarify though for Growth team.